hello-world-mi/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+displayName: Hello World in MI
+description: Hellow World project created using WSO2 Micro Integrator
+componentType: service
+buildPreset: mi
```

Comment:
```suggestion
buildPreset: wso2-mi
```

Reply:
Fixed in dbebb9a56cd9f001df2b9b4cbb98397a9ada986f